widgetssdk/src/main/java/com/glia/widgets/chat/ChatView.kt

```diff
@@ -528,7 +528,11 @@ class ChatView(context: Context, attrs: AttributeSet?, defStyleAttr: Int, defSty
 
     private fun requestOpenUri(uri: Uri) {
         Intent(Intent.ACTION_VIEW, uri).addCategory(Intent.CATEGORY_BROWSABLE).also {
-            context.startActivity(it)
+            if (it.resolveActivity(context.packageManager) != null) {
+                context.startActivity(it)
+            } else {
+                showToast(context.getString(R.string.glia_dialog_unexpected_error_title))
```

Comment:
I would suggest to ask @alina-kauts to help with composing a specific error message that can give a visitor and integrator a clear understanding what's wrong. Something like "Application that can handle action not found".

Reply:
I can add Logs for developers, but I don't think we need to expose such information to the visitors, that's why I thought to use this common message, but it is also okay to not show anything for visitors in this case.